### ISSUE TITLE
Open Popup: Cannot be "embedded at the right of the window"

### ISSUE THREAD

**Issue by JStratil** [oaubert/advene#88]
The option "embedded at the right of the window" for "open popups in the preferences shows no result, i.e. no popup appears.

**Comment by oaubert** [comment]
Are you sure that no popup appears? Is it not that the view area is completely hidden (there is a handle that allows to resize the different areas)?

**Comment by JStratil** [comment]
The option to display a popup right to the video in the view area functions very well. When choosing "as a popup window" a second popup-window besides the opened project appears. However when choosing "embedded at the right of the window", I couldn't find the popup window. Should it appear as if one has two projects open (like two tabs in a browser)? The last behavior happened to a colleague working on Mac (with the Mac-version) when opening any new view or popup – also when other preferences for popups where selected. While trying to reproduce that behavior I stumbled upon the issue that the popup-option "embedded at the right of the window" doesn't seem to work on by setup (Mac and linux). (Whether there is connection to the issue of my colleague is of course not clear. In any case I wanted to let you know.) I hope this description is helpful.

**Comment by oaubert** [comment]
Just to be sure: the "Embedded at the right of the window" means, when looking at https://github.com/oaubert/advene/wiki/AdveneUserGuide#a-tour-of-the-interface, "Open new views in the fareast view location". Is the "fareast view location" pane open in your Advene configuration? It sometimes happens that it is completely hidden (the handle between East view location and Fareast view locaation fully to the right)

**Comment by JStratil** [comment]
Yes, the fareast location was hidden. Thank you! I assumed a different behavior in light of my colleagues issue. But the popup-option works fine.
